python/xgrammar/matcher.py

```diff
@@ -400,3 +400,122 @@ def _debug_print_internal_state(self) -> str:
             The internal state of the matcher.
         """
         return self._handle._debug_print_internal_state()
+
+    @staticmethod
+    def batched_fill_next_token_bitmask(
+        matchers: List["GrammarMatcher"],
+        bitmask: ArrayLike,
+        index: int = 0,
+        max_threads: int = 16,
+        debug_print: bool = False,
+    ) -> List[bool]:
+        """Fill the next token bitmask for multiple matchers.
+
+        Parameters
+        ----------
+        matchers : List[GrammarMatcher]
+            The list of matchers to fill the bitmask for.
+
+        bitmask : ArrayLike
+            The bitmask to fill.
+
+        index : int, default: 0
+
+        max_threads : int, default: 16
+            The maximum number of threads to use for filling the bitmask.
+                    index : int, default: 0
+            The batch id of the bitmask.
+
+        debug_print : bool, default: False
+            Whether to print information about generated bitmask. Helpful for debugging.
+                    Returns
+
+        Returns
+        -------
+        need_apply : List[bool]
+            Whether the bitmask need to be applied (not all-true). An optimization: if False,
+            this means the bitmask is already all-true, so no need to apply it.
+
+        Raises
+        ------
+        RuntimeError
+            If the bitmask is invalid (not on CPU, not int32, shape mismatch).
+
+            If the recursion depth is exceeded.
```

Comment:
remove it (also in FillNextTokenBitmask)

In docs:

These methods will raise RuntimeError:
* JSON Schema Converter

Before version xxx: these methods will raise RuntimeError:
* ...
* ....